main.js

```diff
@@ -89,29 +89,35 @@ function getAll(){
 // main ranking function,
 function rank(players, games){
     function adjust(ps, game){
-        var winners = game.winner === 'l'? game.playersL : game.playersR
-        var losers = game.winner === 'r'? game.playersL : game.playersR
-        var allParticipants = [...winners, ...losers]
-        var winners = ps
-            .filter(p=>winners.includes(p.id))
-            .map(p=>({rank: 1, id:p.id, skill:p.skill}))
-        var losers = ps
-            .filter(p=>losers.includes(p.id))
-            .map(p=>({rank: 2, id:p.id, skill:p.skill}))
-        var toAdjust = [...winners, ...losers]
-        trueskill.AdjustPlayers(toAdjust)
-        return ps.map(p=>({
-            id: p.id,
-            skill: allParticipants.includes(p.id)?
-                toAdjust.filter(o=>o.id===p.id)[0].skill : p.skill
-        }))
+        var rankL, rankR;
```

Comment:
in a spirit of error message free minimalism, could replace all this with `var [rankL, rankR] = game.winner === 'l'? [1, 2] : [2, 1]`